recipes/nifti_clib/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "nifti_clib" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NIFTI-Imaging/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 62655d8457bd04ffd80cf2f07fe8c6aac2b8b00a69fbe71f399e3d9623128630
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+  run:
+    - expat
+    - zlib
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/nifti1.h
+    - test -f ${PREFIX}/lib/libniftiio.a
```

Comment:
Does this only provide a static library? Or also a shared one?

If there is a shared one it might make sense to include run_exports